FordLabs/PeopleMover
# View only by space

## Issue
Resolves 3 57

## What was done
- [x] Add read only toggle at the space level to allow spaces to individually turn on and off read only

## How to test
1. Use the toggle to turn on read only in share access modal
2. Make sure someone without access to space has read only view
3. Use toggle to turn off read only
4. Make sure that same person now does not have read only access to the space
